File: src/blocks/types/stone.go
package blocks

// StoneBlock is a block of rock
struct StoneBlock {
    Block
    lifespan, health f32
}

// newStoneBlock creates a new GrassBlock and returns it
fn newStoneBlock() *StoneBlock {
	return &StoneBlock{
        lifespan: 100,
        health: 100,
    }
}

File: src/blocks/types/stone.rs
/// StoneBlock is a block of rock
struct StoneBlock {
    lifespan: f32,
    health: f32,
}

impl StoneBlock {
    /// Creates a new GrassBlock and returns it
    fn new() -> Self {
        Self {
            lifespan: 100,
            health: 100,
        }
    }
}

impl Block for StoneBlock {

}

